services/sqs/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+log4j.rootLogger = DEBUG, A1
+log4j.appender.A1 = org.apache.log4j.ConsoleAppender
+log4j.appender.A1.layout = org.apache.log4j.PatternLayout
+
+# Print the date in ISO 8601 format
+log4j.appender.A1.layout.ConversionPattern = %d [%t] %-5p %c - %m%n
+
+# Adjust to see more / less logging
+#log4j.logger.com.amazonaws.ec2=DEBUG
+
+# HttpClient 3 Wire Logging
+log4j.logger.httpclient.wire=DEBUG
+
+# HttpClient 4 Wire Logging
+ log4j.logger.org.apache.http=DEBUG
```

Comment:
Todo to remove this file at some point? Even just in the test directory a bit sketchy to have properties file that have wire logging enabled around

Reply:
We -- can -- we have them in a bunch of our integ-tests and they have been handy. I'd love it if it was something that we could configure on a run-by-run basis rather than having to commit it to source...

Reply:
Removed.